ocp_resources/console_plugin.py

```diff
@@ -0,0 +1,10 @@
+from ocp_resources.resource import Resource
+
+
+class ConsolePlugin(Resource):
+    """
+    ConsolePlugin object.
+    """
+
+    api_group = Resource.ApiGroup.CONSOLE_OPENSHIFT_IO
+    api_version = Resource.ApiVersion.V1ALPHA1
```

Comment:
api_version should be set automatically.

Reply:
I will remove this.